template.pkr.hcl

```diff
@@ -42,47 +70,49 @@ source "amazon-ebs" "ubuntu" {
   }
   ssh_username = "ubuntu"
   tags = {
-    Name       = "Coder"
-    OS_Version = "Ubuntu"
+    name       = "Coder"
+    os         = "Ubuntu"
   }
 }
 
 build {
   name = "coder"
   sources = [
-    "source.amazon-ebs.ubuntu"
+    "source.amazon-ebs.ubuntu",
+    "source.googlecompute.ubuntu"
```

Comment:
commenting this out so that the existing AWS action doesn't fail. Please uncoment when youa dd the GCP market place action.